__tests__/login/oidc/ClientRegistrar.spec.ts

```diff
@@ -66,12 +75,15 @@ describe("ClientRegistrar", () => {
         ) as unknown) as Response
         /* eslint-enable @typescript-eslint/camelcase */
       );
-      const clientRegistrar = getClientRegistrar();
+      const clientRegistrar = getClientRegistrar({
+        storage: EmptyStorageUtilityMock
+      });
       const registrationUrl = new URL("https://idp.com/register");
       expect(
         await clientRegistrar.getClient(
           {
-            redirect: new URL("https://example.com")
+            sessionId: "mySession",
+            redirectUrl: new URL("https://example.com").toString()
```

Comment:
Kinda strange to see this - I understand why, but given the property is meant to be a string now (I'd still prefer the original URL object, but let's not get into that now!), but this code should be just `redirectUrl: "https://example.com"`.
It also raises the question of whether we (i.e. Inrupt) should use a consistent test vocabulary across all our code (both Java and JavaScript, and C#, etc.), that defines example WebID's and Resource IRIs and Container IRIs, etc. We have an initial Test Vocab now here: https://github.com/inrupt/lit-vocab/blob/master/inrupt-rdf/Common/CopyOfVocab/inrupt-test.ttl, and in fact using it would help make this code more readable I think - i.e. what (expressed in English) would we normally expect the 'redirectUrl' to actually be - i.e. is it the WebID, or the Root Container of the Pod, or something else (i.e. "https://example.com" does really tell me anything)...???

Reply:
Ooh I see the issue, there is an internal misalignment: some URLs are typed as such, and some other are typed as `string`, as here: 
```typescript
export interface IRegistrarOptions {
  sessionId: string;
  clientId?: string;
  clientSecret?: string;
  clientName?: string;
  redirectUrl?: string;
}
```

Reply:
Fixed in ab0a102d14793ea02b4a99a62ec835f32d1eba80